examples/gno.land/p/demo/memeland/memeland.gno

```diff
@@ -26,15 +26,15 @@ type Post struct {
 }
 
 type Memeland struct {
-	*ownable.Ownable
+	ownable.Transferrable
```

Comment:
You're not exposing any transferable helper, so using the Ownable interface is sufficient.

Reply:
sure, i realized it

Reply:
the value returned from owner.New() is currently a `transferrable`, which implements the `Transferrable` interface. Using the `Owner` interface here is not appropriate

Reply:
Can someone help me explain?

Reply:
I understand Go interfaces, but it seems my explanation was not correct. However, I need to confirm with you that there is a call to TransferOwnership in /p/demo/memeland